### PR TITLE
Added integration test documentation. (Issue #626)

### PR DESCRIPTION
**Pull request by antonini**
Added integration test documentation. (Issue #626)

### CODE REVIEW

**Comment by karato** [comment]
@antonini Thanks, I will find someone to review this PR soon

**Comment by karato** [comment]
@ggajos do this review

File: README.md [hunk 1]
```diff
@@ -63,3 +65,20 @@ $ mvn clean install -Phit-refresh -Dport=8080
 ```
 
 In a minute the site is ready at `http://localhost:8080`
+
+### Integration tests
+
+It is highly recommended to run integration tests to guarantee that your changes didn't broke any other part of the system.
+Execute the following steps to execute integration tests:
```
Comment (ggajos) [inline README.md]: @antonini let's avoid using same word `Execute` in same sentence.

File: README.md [hunk 2]
```diff
@@ -63,3 +65,20 @@ $ mvn clean install -Phit-refresh -Dport=8080
...
+
+1. Open your browser and its browser's console and go to network view;
```
Comment (ggajos) [inline README.md]: @antonini let's avoid using same word browser in same sentence.

File: README.md [hunk 3]
```diff
@@ -63,3 +65,20 @@ $ mvn clean install -Phit-refresh -Dport=8080
 ```
 
 In a minute the site is ready at `http://localhost:8080`
+
+### Integration tests
+
+It is highly recommended to run integration tests to guarantee that your changes didn't broke any other part of the system.
```
Comment (ggajos) [inline README.md]: @antonini let's replace did not broke with `will not break`

**Comment by ggajos** [comment]
@antonini few remarks

**Comment by antonini** [comment]
@ggajos Fixed and ready to review.

**Comment by ggajos** [comment]
@rultor merge please

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by antonini** [comment]
@dmzaytsev I don't know if you have access to rultor. But it is running out of space.

    fatal: write error: No space left on device fatal: index-pack failed

**Comment by dmzaytsev** [comment]
@antonini I have created the ticket https://github.com/yegor256/rultor/issues/906 
don't worry we will  wait until the issue will be resolved

**Comment by antonini** [comment]
@dmzaytsev Ok. My question is: Does it going to be considered on my performance until that issue be solved?

**Comment by dmzaytsev** [comment]
@antonini I think it will be resolved soon.
If no you can inform the PM right in the ticket about the delay. In this case the PM will put the ticket on hold

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by karato** [comment]
@ypshenychka please, let us know what do you think about this ticket, according to our [QA rules](http://www.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor please deploy

**Comment by ypshenychka** [comment]
@karato Quality is good.

**Comment by karato** [comment]
> @karato Quality is good.

@ypshenychka thanks!

**Comment by karato** [comment]
@ggajos since quality is good, 10 mins paid to @dmzaytsev (our architect), transaction `61633792`

**Comment by karato** [comment]
@ggajos I added 10 mins to @ypshenychka (for QA review) in transaction 61641635

Thanks for your contribution, **17 mins** was added to your account, payment ID is `AP-98X22483R26011429`, 47 hours and 5 mins spent total

review comments (c=2) added as a bonus

added +17 to your rating, now it is equal to [+742](http://www.netbout.com/b/35094?open=rating)

**Pull request merged**